code/modules/mob/mob_movement.dm

```diff
@@ -137,8 +137,12 @@
 
 	if(isturf(mob.loc))
 
+
+		var/turf/T = mob.loc
 		move_delay = world.time//set move delay
 
+		move_delay += T.slowdown

```

Comment:
Incorrect @phil235 , I was going to report this myself, but it is below an ```if(isturf(mob.loc))``` check